webapi/middleware.go

```diff
@@ -6,20 +6,24 @@ package webapi
 
 import (
 	"bytes"
+	"encoding/base64"
 	"errors"
+	"fmt"
 	"io"
 	"net/http"
 	"strings"
+	"time"
 
-	"github.com/decred/dcrd/blockchain/stake/v4"
-	"github.com/decred/dcrd/chaincfg/chainhash"
 	"github.com/decred/vspd/rpc"
 	"github.com/gin-gonic/gin"
 	"github.com/gin-gonic/gin/binding"
 	"github.com/gorilla/sessions"
 	"github.com/jrick/wsrpc/v2"
 )
 
+// Message is declared as a const so other projects can easily reference it.
+const Message = "I want to check vspd ticket status for ticket %s on VSP %s on block window %d."
```

Comment:
```suggestion
// TicketSearchMessageFmt is the format for the message to be signed in order to search for a ticket using the vspd frontend.
const TicketSearchMessageFmt = "I want to check vspd ticket status for ticket %s on VSP %s on block window %d."
```

Cap the comment at max 80 chars per line.